### DISCUSSION TITLE
-/-

### DISCUSSION THREAD

**Discussion by BornToBeRoot** [renovatebot/renovate#33095] [Suggest an Idea]
If you automatically close issues/bug reports and then restrict access to the repo, you probably don't want any interaction with the community. Bye.

**Comment by rarkins** [comment]
Should these continue to be separate datasources, or is the potential to combine them if we make them more generic?

**Reply by rarkins** [reply]
How many such "different" datasources would be likely to have in future from AWS? e.g. today we have 3, but what's the maximum number?
